examples/contrib/epidemiology/regional.py

```diff
@@ -149,6 +150,7 @@ def main(args):
     parser.add_argument("--single", action="store_false", dest="double")
     parser.add_argument("--rng-seed", default=0, type=int)
     parser.add_argument("--cuda", action="store_true")
+    parser.add_argument("--jit", action="store_true")
```

Comment:
shouldn't this be on by default?

Reply:
Good question. How about we enable it on by default in the example files, but off by default in the `CompartmentalModel` class? I just worry that `jit_compile=True` disables validation, and that if we optimize for new users we should enable validation by default. The example scripts use predefined models and don't really need validation, whereas `CompartmentalModel` will be the base class for new models.

Reply:
yeah that sounds like a reasonable middle ground

Reply:
done.